tests/integration/go.mod

```diff
@@ -13,7 +13,7 @@ go 1.22.2
 require (
 	cosmossdk.io/math v1.3.0
 	github.com/cometbft/cometbft v0.38.7
-	github.com/cosmos/cosmos-sdk v0.50.6
+	github.com/cosmos/cosmos-sdk v0.50.7
```

Comment:
GoLand will not stop updating this.... SEND HELP

Reply:
changelog is [here](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.50.7), doesn't look like anything super relevant to us, so this is safe+